nbdime/webapp/nbdimeserver.py

```diff
@@ -173,7 +212,7 @@ def build_arg_parser():
     """
     description = 'Web interface for Nbdime.'
     parser = ArgumentParser(description=description)
-    parser.add_argument('-p', '--port',
+    parser.add_argument('-p', '--port', default="8888",
```

Comment:
Default should be an integer, not string of integer. Then argparse will do the right type casting.